lib/user/user-stack.ts

```diff
@@ -0,0 +1,80 @@
+import { Construct } from 'constructs';
+import {
+  OAuthScope,
+  UserPool,
+  UserPoolClient,
+  UserPoolClientIdentityProvider,
+  VerificationEmailStyle,
+} from 'aws-cdk-lib/aws-cognito';
+import { RemovalPolicy } from 'aws-cdk-lib';
+
+export class UserStack extends Construct {
+  public userPool: UserPool;
+  public userPoolClient: UserPoolClient;
+
+  public constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    this.userPool = this.createUserPool();
+    this.userPoolClient = this.createUserPoolClient(this.userPool);
+  }
+
+  public createUserPool(): UserPool {
+    const userPool = new UserPool(this, 'project-user-pool', {
+      userPoolName: 'project-user-pool',
+      selfSignUpEnabled: true,
+      signInAliases: {
+        email: true,
+      },
+      standardAttributes: {
+        givenName: {
+          required: true,
+          mutable: false,
+        },
+        familyName: {
+          required: true,
+          mutable: false,
+        },
+      },
+      removalPolicy: RemovalPolicy.DESTROY,
+      deletionProtection: false,
+      autoVerify: {
+        email: true,
+      },
+      passwordPolicy: {
+        minLength: 8,
+        requireLowercase: true,
+        requireDigits: false,
+        requireSymbols: false,
```

Comment:
why don't we require digits or symbols? Strong passwords are usually a combination of words with numbers

Reply:
@ekertt Why did you make this choice?

Reply:
@Ryukishi I didn't found it that necessary to include this as required, maybe you can tell me if and why this is necessary for our MVP?

Reply:
@ekertt I believe this is necessary for our MVP because we are looking to deliver an MVP that includes complete code, not unsafe code.

Reply:
Or rather code that could harm our users

Reply:
@Ryukishi Good point, will change it now.